models/database.go

```diff
@@ -189,6 +189,22 @@ var databaseSchema = [][]string{
 		`ALTER TABLE agents
 			ADD COLUMN table_count INTEGER`,
 	},
+
+	7: {
+		`ALTER TABLE agents ADD COLUMN service_id VARCHAR CHECK (service_id <> '')`,
+		`UPDATE agents SET service_id=agent_services.service_id 
+			FROM agent_services 
+			WHERE agent_services.agent_id = agents.agent_id`,
+		`ALTER TABLE agents ADD COLUMN node_id VARCHAR CHECK (node_id <> '')`,
+		`UPDATE agents SET node_id=agent_nodes.node_id 
+			FROM agent_nodes
+			WHERE agent_nodes.agent_id = agents.agent_id`,
+		"DROP TABLE agent_services",
+		"DROP TABLE agent_nodes",
+		"ALTER TABLE agents ADD CONSTRAINT not_service_id_and_node_id CHECK ((service_id IS NULL) OR (node_id IS NULL))",
+		"ALTER TABLE agents ADD CONSTRAINT service_id_fk FOREIGN KEY (service_id) REFERENCES services(service_id) ON DELETE CASCADE",
+		"ALTER TABLE agents ADD CONSTRAINT node_id_fk FOREIGN KEY (node_id) REFERENCES nodes(node_id) ON DELETE CASCADE",
```

Comment:
I will remove `ON DELETE CASCADE` – we should not delete data silently, only when `force` flag / API field is used, and in that case we should do it manually.